src/OrchardCore.Modules/OrchardCore.ContentFields/Indexing/SQL/Migrations.cs

```diff
@@ -807,5 +807,14 @@ public async Task<int> UpdateFrom4Async()
 
             return 5;
         }
+
+        public async Task<int> UpdateFrom5Async()
+        {
+            await SchemaBuilder.AlterIndexTableAsync<LinkFieldIndex>(table => table
```

Comment:
This is needed in `CreateAsync` too.

Reply:
Its not needed. `CreateAsync` returns 3. From there we do other steps including the new step. If we add the column during create, then this step will through an exception

Reply:
[`CreateAsync` returns 5](https://github.com/OrchardCMS/OrchardCore/blob/8e8f2d78ddbe5106d9fcd0f371ffb28c2cf116f4/src/OrchardCore.Modules/OrchardCore.ContentFields/Indexing/SQL/Migrations.cs#L437) for the same reason.

So, we need to add the column there too, and update its return value to `6`.

Reply:
I don't know why I saw it return 3 not return 5 :) I changed it.